src/LanguageServer.Engine/Documents/MasterProjectDocument.cs

```diff
@@ -22,7 +23,7 @@ public class MasterProjectDocument
         /// <summary>
         ///     Sub-projects (if any).
         /// </summary>
-        readonly Dictionary<Uri, SubProjectDocument> _subProjects = new Dictionary<Uri, SubProjectDocument>();
+        readonly ConcurrentDictionary<Uri, SubProjectDocument> _subProjects = new ConcurrentDictionary<Uri, SubProjectDocument>();
```

Comment:
I don't remember if I asked you previously: what is your opinion on `new()` syntax? Using it in places like this can reduce the amount of changes in case like this one, but there are also perfectly legit arguments against `new()` syntax as well

Reply:
I’m ok with either var or new() as long as the type name appears somewhere on the line. Will update!